06-Modelling-Rigid-Bodies/01-Introduction-to-Rigid-Bodies.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<p>Definition of orientation is confusing. I'm not sure how to interpret "rotates from body to global coordinates"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Does this make more sense:</p><p> </p><p>a quaternion that [rotates](https://en.wikipedia.org/wiki/Rotation_(mathematics)) the rigid body about the central particle.</p>